tests/sentry/incidents/test_logic.py

```diff
@@ -836,6 +836,25 @@ def test(self):
         assert self.alert_rule.projects.all().count() == 2
         assert self.alert_rule.projects.all()[0] == updated_projects[0]
 
+    @mock.patch(
+        "sentry.workflow_engine.migration_helpers.alert_rule.dual_update_migrated_alert_rule"
+    )
+    def test_dual_update(self, mock_dual_update):
+        # test that we call the ACI dual update helpers-will be removed after dual write period ends
+        name = "hojicha"
+
+        updated_rule = update_alert_rule(
+            self.alert_rule,
+            name=name,
+        )
+        assert self.alert_rule.id == updated_rule.id
+        assert self.alert_rule.name == name
+
+        assert mock_dual_update.call_count == 1
+        call_args = mock_dual_update.call_args_list[0][0]
+        assert call_args[0] == self.alert_rule
+        assert call_args[1]["name"] == name
```

Comment:
would it be possible to fetch the relevant ACI models (in this case I think it's the detector's name that would be changed) and assert on those changes? you could call `migrate_alert_rule` first and then update it